apps/dashboard/src/components/workflow-editor/steps/in-app-editor.tsx

```diff
@@ -0,0 +1,113 @@
+import { RiEdit2Line, RiInformationFill, RiPencilRuler2Line } from 'react-icons/ri';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useFormContext } from 'react-hook-form';
+import * as z from 'zod';
+
+import { Button } from '@/components/primitives/button';
+import { Separator } from '@/components/primitives/separator';
+import { SheetClose } from '@/components/primitives/sheet';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { Notification5Fill } from '@/components/icons';
+import { AvatarPicker } from '@/components/primitives/form/avatar-picker';
+import { Input, InputField } from '@/components/primitives/input';
+import { Textarea } from '@/components/primitives/textarea';
+import { workflowSchema } from '../schema';
+import { ActionPicker } from '../action-picker';
+import { URLInput } from '@/components/primitives/url-input';
+import { urlTargetTypes } from '@/utils/url';
+import { RedirectTargetEnum } from '@novu/shared';
+
+const tabsContentClassName = '!mt-0 h-full w-full px-3 py-3.5';
+
+export const InAppEditor = () => {
+  const navigate = useNavigate();
+  const { formState } = useFormContext<z.infer<typeof workflowSchema>>();
+
+  return (
+    <Tabs defaultValue="editor" className="flex h-full flex-1 flex-col">
+      <header className="flex flex-row items-center gap-3 px-3 py-1.5">
+        <div className="mr-auto flex items-center gap-2.5 text-sm font-medium">
+          <RiEdit2Line className="size-4" />
+          <span>Configure Template</span>
+        </div>
+        <TabsList className="w-min">
+          <TabsTrigger value="editor" className="gap-1.5">
+            <RiPencilRuler2Line className="size-5 p-0.5" />
+            <span>Editor</span>
+          </TabsTrigger>
+          <TabsTrigger value="preview" className="gap-1.5">
+            <Notification5Fill className="size-5 p-0.5" />
+            <span>Preview</span>
+          </TabsTrigger>
+        </TabsList>
+
+        <SheetClose
+          className="ring-offset-background focus:ring-ring rounded-sm opacity-70 focus:outline-none focus:ring-2 focus:ring-offset-2 disabled:pointer-events-none"
+          onClick={() => {
+            navigate('../', { relative: 'path' });
```

Comment:
Do we need the `<SheetClose />` here? Does it actually animate or is it instant?